android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
Ended up writing the S3 upload in Java, didn't find any library to help.

Getting the raw image in base64 and uploading from RN could work too, but I had issues manipulating large files in RN land. Packages like [react-native-fs](https://github.com/johanneslumpe/react-native-fs) only work with path too.